src/clin_nlp_metrics/metrics.py

```diff
@@ -142,12 +153,23 @@ def from_medcattrainer(data: dict) -> "Dataset":
 
             for annotation in doc["annotations"]:
                 if not annotation["deleted"]:
+                    qualifiers = []
+
+                    for qualifier in annotation["meta_anns"].values():
+                        qualifiers.append(
+                            {
+                                "name": qualifier["name"].title(),
+                                "value": qualifier["value"].title(),
```

Comment:
Is it a problem that medcat has no is_default? Or should we just set it to True?

Reply:
Medcat doesn't know/output what the default is. So we need to add that manually or infer it. Will add later